packages/api/src/generated/schema.graphql

```diff
@@ -2071,6 +2129,8 @@ enum SubscribeErrorCode {
 }
 
 input SubscribeInput {
+  isFetchingContent: Boolean
```

Comment:
Is this a state, or is it like "automatically add to library"?

Reply:
I know it is confusing but I meant "automatically add to library"